frontend/src/containers/DocumentList.js

```diff
@@ -134,8 +134,11 @@ class DocumentListContainer extends Component {
      * TODO: This could probably be handled by a combination of
      * middleware reacting to route changes and reducers
      */
+
     if (
-      (nextProps.viewId !== nextProps.queryViewId && nextProps.queryViewId) || // for the case when you applied a filter and come back via browser back button
+      (nextProps.viewId !== nextProps.queryViewId &&
+        nextProps.queryViewId &&
+        !this.renderedSuccessfuly) || // for the case when you applied a filter and come back via browser back button
```

Comment:
Just set this to false when initializing the component and move the comment.

Reply:
It will resolve to false since is not defined. Do you still want to add it ? Or just formatting the code would be sufficient ?

Reply:
It's just for the sake of readability.

Reply:
ok